### PR TITLE
block and chain 3d render

### PR DESCRIPTION
**Pull request by ghost**
I tried to show 3d items of block and chain

original issues:https://github.com/TeamTwilight/twilightforest/issues/256

### CODE REVIEW

**Review by Drullkus — CHANGES_REQUESTED** [review]
Tbh I'm not fully sold on this implementation.

Perhaps you can change the scope of this PR to actually changing the way the Block-And-Chain works.

Part of the change is making the Block And Chain sensitive to whether the player has an item in their other hand or not. The Block And Chain will not be usable if the player is also holding another item (`.isEmpty()`).

If this is happening, then Block And Chain will render as normal in the hand that is holding the item.

If the player has their other hand empty, then the player should be able to use the Block And Chain. They should also have their arm using the item raised like they will for 1.13's spear. (This though may best be implemented post-1.13...) The Block of the Block And Chain should be swinging around above their heads the same way it does for the Goblin.

**Comment by ghost** [comment]
Changed the behavior of Block And Chain to work when one hand is free

File: src/main/java/twilightforest/item/ItemTFChainBlock.java [hunk 1]
```diff
@@ -43,6 +43,14 @@ public float apply(ItemStack stack, @Nullable World worldIn, @Nullable EntityLiv
 				return getThrownUuid(stack) != null ? 1 : 0;
 			}
 		});
+
+		this.addPropertyOverride(TwilightForestMod.prefix("cannotthrow"), new IItemPropertyGetter() {
```
Comment (Drullkus) [inline src/main/java/twilightforest/item/ItemTFChainBlock.java]: I'd change `cannotthrow` to `cannot_throw`, same for in the item json file

**Review by Drullkus — CHANGES_REQUESTED** [review]
Just one last thing, and then I'll pull this in, pending quad's review too :)

**Comment by Tamaized** [comment]
Closing this as we're discussing a completely different implementation than this.

**Pull request closed without merging**